Adapted from
https://github.com/Mindwerks/wildmidi/commit/a128266be48f2e2fabcf308232f88c2bbaa03677

Replace mindwerks.net home page with the github project page in the package.
---
diff --git a/README.md b/README.md
index 4f33e7e..e05912a 100644
--- a/README.md
+++ b/README.md
@@ -8,7 +8,7 @@ applications that wish to include MIDI file playback.
 
 Version: 0.4.5
 Licenses: GPLv3+ and LGPLv3
-Website: http://www.mindwerks.net/projects/wildmidi
+Website: https://github.com/Mindwerks/wildmidi
 
 PLATFORMS:
 
diff --git a/include/config.h.cmake b/include/config.h.cmake
index 65ab0b8..3f52ae7 100644
--- a/include/config.h.cmake
+++ b/include/config.h.cmake
@@ -4,7 +4,7 @@
 #define PACKAGE "wildmidi"
 
 /* Define to the home page for this package. */
-#define PACKAGE_URL "http://www.mindwerks.net/projects/wildmidi/"
+#define PACKAGE_URL "https://github.com/Mindwerks/wildmidi"
 
 /* Define to the address where bug reports for this package should be sent. */
 #define PACKAGE_BUGREPORT "https://github.com/Mindwerks/wildmidi/issues"
diff --git a/src/wildmidi.pc.in b/src/wildmidi.pc.in
index 3d8e84f..1a73284 100644
--- a/src/wildmidi.pc.in
+++ b/src/wildmidi.pc.in
@@ -8,7 +8,7 @@ includedir=${exec_prefix}/@CMAKE_INSTALL_INCLUDEDIR@
 Name: WildMIDI
 Description: software synthesizer library
 Version: @WILDMIDI_VERSION@
-URL: https://www.mindwerks.net/projects/wildmidi/
+URL: https://github.com/Mindwerks/wildmidi
 
 Libs: -L${libdir} -lWildMidi
 Libs.private: -lm 
